server/server.go

```diff
@@ -1,154 +0,0 @@
-package main
-
-import (
-	"fmt"
-	"io/ioutil"
-	"log"
-	"strconv"
-	"net/http"
-	"encoding/json"
-	"database/sql"
-
-	"github.com/willf/bloom"
-	"github.com/blics18/SendGrid/client"
-	_ "github.com/go-sql-driver/mysql"
-	
-)
-
-type bloomFilter struct {
-	filter *bloom.BloomFilter
-}
-
-func createBloomFilter() *bloomFilter {
-	return &bloomFilter{
-		filter: bloom.New(20*uint(1000), 5),
-	}
-}
-
-func (bf *bloomFilter) populateBF(w http.ResponseWriter, r *http.Request) {
-	if r.Body == nil {
-		w.Write([]byte("Could not read the body of the request"))

```

Comment:
this will register as http.StatusOK. You need to write the header you want here, and if it is an error case, I would return from the function